docs/revsets.md

```diff
@@ -187,6 +187,23 @@ You can use parentheses to control evaluation order, such as `(x & y) | z` or
     * `D..B` ⇒ `{}` (empty set)
     * `(C|B)..(C|B)` ⇒ `{}` (empty set)
 
+### Comparison of the operators `::` and `..`
+
+The `x::y` operator behaves like an "inclusive range" operator.
+
+The `x..y` operator behaves like a set-difference operator, removing `::x` from
```

Comment:
We have `~` for set difference, so the first part here is perhaps a bit confusing.

What do you think about the following?

> `x..y` is the set of revisions introduced from `x` to `y`.